db/init.sql

```diff
@@ -78,4 +78,15 @@ CREATE TABLE IF NOT EXISTS performance.alert (
     heap_size_degraded integer,
     query_rows_degraded integer,
     soql_queries_degraded integer
+);
+
+CREATE TABLE IF NOT EXISTS performance.test_info (
+    id serial PRIMARY KEY,
+    create_date_time timestamp without time zone DEFAULT now() NOT NULL,
+    update_date_time timestamp without time zone DEFAULT now() NOT NULL,
+    flow_name text,
+    action text,
+    product text,
+    functional_area text,
+    description  text
 );
```

Comment:
Should have a migrations V3_test_info.sql script with just this table in to use, though technically running this file would work as well given the `IF NOT EXISTS`.